include/triton/Dialect/TritonGPU/IR/Dialect.h

```diff
@@ -89,9 +89,8 @@ SmallVector<unsigned> getCTAOrder(Attribute layout);
  * (3) In the implementation of emitIndices, ShapePerCTATile will
  *     be replicated or wraped to fit ShapePerCTA.
  */
-SmallVector<unsigned>
-getShapePerCTATile(Attribute layout,
-                   ArrayRef<int64_t> tensorShape = ArrayRef<int64_t>());
+SmallVector<unsigned> getShapePerCTATile(Attribute layout,
+                                         ArrayRef<int64_t> tensorShape);
```

Comment:
Why is this default value of `tensorShape` removed? It seems that in many cases such as `BlockedEncodingAttr`, `tensorShape` is not required.

Reply:
if using default parameter, I guess it is easy for user forgot provide tensorShape for some encoding.